### ISSUE TITLE
Add VS2019 Support

### ISSUE THREAD

**Issue by Hedreon** [0xd4d/dnSpy#1404]
it's been a while since DnSpy added it's latest VS. (used to be VS 2017, now it's VS 2019)

**Comment by 0xd4d** [comment]
I assume you mean using the latest Roslyn (C# 8 support).

**Comment by Hedreon** [comment]
Issue edited, Please read again.

**Comment by Hedreon** [comment]
Yeah...? add that support and add the VS 2019 option into the save project form.

**Comment by CAA42069** [comment]
![alt text](https://images.pornpics.com/1280/201804/16/4265856/4265856_039_ce70.jpg)

**Comment by Hedreon** [comment]
This is kind of a long time ago, should I close this now?
